corehq/apps/style/static/style/less/components/layouts.less

```diff
@@ -87,3 +87,35 @@
 #report-content {
     clear: both;
 }
+
+.top10 {
+  margin-top: 10px;
+}
+
+.top20 {
+  margin-top: 20px;
+}
+
+.top1em {
+  margin-top: 1em;
+}
```

Comment:
This feels better to me; it allows other people to use the same classes for layout, it seems like a better place to put these and creates a place to add more if, for some reason, you need a 5px margin, or whatever.